airgun/views/os.py

```diff
@@ -9,6 +9,9 @@
 class OperatingSystemView(BaseLoggedInView):
     title = Text("//h1[text()='Operating systems']")
     new = Text("//a[contains(@href, '/operatingsystems/new')]")
+    delete = GenericLocatorWidget(
```

Comment:
Still `GenericLocatorWidget`? Won't `Text` work in this case?

Reply:
So, I have incorporated the changes (I hope) except for this one. I still think I should use most generic class possible and `GenericLocatorWidget` is a first one with `.click()` method. But I do not have strong opinion.

Reply:
Neither do i. We should probably think of some custom `Button` widget for our needs later.